R/pkg/R/functions.R

```diff
@@ -2043,34 +2033,6 @@ setMethod("countDistinct",
             column(jc)
           })
 
-#' @details
-#' \code{concat}: Concatenates multiple input columns together into a single column.
-#' If all inputs are binary, concat returns an output as binary. Otherwise, it returns as string.
-#'
-#' @rdname column_string_functions
-#' @aliases concat concat,Column-method
-#' @examples
-#'
-#' \dontrun{
-#' # concatenate strings
-#' tmp <- mutate(df, s1 = concat(df$Class, df$Sex),
-#'                   s2 = concat(df$Class, df$Sex, df$Age),
-#'                   s3 = concat(df$Class, df$Sex, df$Age, df$Class),
-#'                   s4 = concat_ws("_", df$Class, df$Sex),
-#'                   s5 = concat_ws("+", df$Class, df$Sex, df$Age, df$Survived))
```

Comment:
also what happen to all other examples?

Reply:
Since now I put concat in the Collection functions section, I am not sure if I should keep the column_string_functions examples there. I will add them back if you prefer to keep these examples.

Reply:
yes, let's not lose example.
we should add concat to the column_collection_functions
and concat_ws to column_string_functions